x-pack/heartbeat/monitors/browser/synthexec/synthexec.go

```diff
@@ -263,7 +272,7 @@ func runCmd(
 	// Close mpx after the process is done and all events have been sent / consumed
 	go func() {
 		err := <-cmdDone
-		jsonWriter.Close()
+		_ = jsonWriter.Close()
```

Comment:
Any particular reason for us to do this assignment? Maybe it's my lack of familiarity with Go, but it doesn't seem it makes a difference in this case.

Reply:
It makes intellij's linter happier. `_ = ` means discard this. It's the only assignment that never uses `:=` because nothing is actually assigned. See https://www.geeksforgeeks.org/what-is-blank-identifierunderscore-in-golang/

TBH, I'm not sure I should be changing this vs. changing my intellij settings but it's certainly not harmful.

Reply:
to be clear, the linter complains about ignored error values, since go doesn't have exceptions